src/helpers/moveToIndex.ts

```diff
@@ -0,0 +1,15 @@
+export const moveToIndex = <T>(
+  items: T[],
+  itemToMoveIndex: number,
```

Comment:
Do you want the condition argument here, @ogroppo? It might lead to unwanted behavior if the array item is duplicated.

Reply:
ah no, risky indeed, just index is fine